scripts/monitors/check_wsgi.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python2.7
```

Comment:
@jasonthomas is `/usr/bin/env python` the expected py2.7 interpreter on production?

To be honest I'm not even sure this script is used at all